doc/tutorials/sagemaker/sme_introduction.ipynb

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
<p>I guess, that is from the existing notebooks, or?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>